.chloggen/span-resource-name.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: tags
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: The tag resolves the operation display value when viewed in the Datadog backend
```

Comment:
As a reader I don't quite understand this note. I'd rephrase as `Add a config span_name_as_resource_name to ... that can do xyz`